frontend/src/views/Pipelines/PipelineList.js

```diff
@@ -118,7 +138,7 @@ const PipelineList = () => {
         dispatch({ type: SET_ERROR, error: e.message })
       );
     }
-  }, [client, filter.page, dispatch, fetchItems]);
+  }, [client, filter, dispatch]);
```

Comment:
with this you are able to remove the fetching in handlekeyinput right? Have you tested that the filter updates fetch the items?

Reply:
Yes - instead of explicitly calling fetchItems for each method I changed so `useEffect` listens to whenever filter has been changed and will call `fetchItems` on any change whether it be term, page, region, tags, or type.

I have tested and works for changing any of the search terms, changing the page, or any of the filters